charts/nx-cloud/templates/_scm-helpers.tpl

```diff
@@ -99,5 +26,26 @@
       name: {{ .Values.secret.name }}
       key: {{ .Values.secret.githubPrivateKey }}
     {{- end }}
+    {{- if .Values.secret.githubWebhookSecret }}
+- name: NX_CLOUD_GITHUB_WEBHOOK_SECRET
+  valueFrom:
+    secretKeyRef:
+      name: {{ .Values.secret.name }}
+      key: {{ .Values.secret.githubWebhookSecret }}
+    {{- end }}
+    {{- if .Values.secret.githubAppClientId }}
+- name: NX_CLOUD_GITHUB_APP_CLIENT_ID
+  valueFrom:
+    secretKeyRef:
+      name: {{ .Values.secret.name }}
+      key: {{ .Values.secret.githubAppClientId }}
+    {{- end }}
+    {{- if .Values.secret.githubAppClientSecret }}
+- name: NX_CLOUD_GITHUB_APP_CLIENT_SECRET
+  valueFrom:
+    secretKeyRef:
+      name: {{ .Values.secret.name }}
+      key: {{ .Values.secret.githubAppClientSecret }}
+    {{- end }}
```

Comment:
I'm checking with Mark about these vars:
1. we either need to add them to the docs: https://nx.dev/ci/recipes/enterprise/on-premise/custom-github-app#using-helm
2. or we don't need them in here